src/test/java/com/plaid/client/integration/InstitutionsSearchTest.java

```diff
@@ -17,6 +17,22 @@ public void testSuccess() throws Exception {
     assertSuccessResponse(response);
   }
 
+  @Test
+  public void testSuccessWithIncludeInstitutionDataTrue() throws Exception {
+    Response<InstitutionsSearchResponse> response =
+        client().service().institutionsSearch(new InstitutionsSearchRequest("t").withIncludeInstitutionData(true)).execute();
+
+    assertSuccessResponse(response);
+  }
+
+  @Test
+  public void testSuccessWithIncludeInstitutionDataFalse() throws Exception {
+    Response<InstitutionsSearchResponse> response =
+        client().service().institutionsSearch(new InstitutionsSearchRequest("t").withIncludeInstitutionData(false)).execute();
+
+    assertSuccessResponse(response);
+  }
+
```

Comment:
These two tests look to me like a missed opportunity to reflect what difference setting `true` vs `false` means for the result object.
Update if you have time.